src/lib.rs

```diff
@@ -31,15 +35,10 @@
 //!     features = ["client-reqwest", "reqwest-native-tls"]
 //! }
 //! ```
-//! Rspotify uses [maybe_async][maybe_async]
-//! crate to switch between different HTTP clients to get different features.
-//! In our case, `rspotify` supports async and blocking feature by triggering
-//! `maybe_async` inside `Cargo.toml`, so we don't have maintain two sets of
-//! code which are mostly same. By default, Rspotify uses asynchronous
-//! programming with `async` and `await` by leveraging `reqwest` crate by
-//! using `client-reqwest` feature. And the blocking IO feature can be
-//! enabled to access with `client-ureq` feature, with non-async methods
-//! and the `rustls` TLS:
+//!
+//! [`maybe_async`] internally enables Rspotify to  use both synchronous and
```

Comment:
Could we add reference linked to `maybe_async` crate ?

Reply:
Same as the other. That's enough for a link to the crate, check it out yourself.

Reply:
Get!